env.example

```diff
@@ -14,6 +14,9 @@ HTTPS_PORT=8443
 # System time zone.
 TZ=Europe/Amsterdam
 
+# Public URL for the web service.
+PUBLIC_URL="https://meet.example.com"
```

Comment:
please leave this one commented out, as the user needs to edit it

Reply:
Ok